### PR TITLE
Stabilize layout components

### PR DESCRIPTION
**Pull request by kyledurand**
### WHY are these changes introduced?

Fixes https://github.com/Shopify/polaris/issues/9978

### CODE REVIEW

File: polaris-react/src/components/Text/README.md [hunk 1]
```diff
@@ -18,9 +18,6 @@ keywords:
   - semibold
```
Comment (kyledurand) [inline polaris-react/src/components/Text/README.md]: @yurm04 can we remove this readme file? I don't think we're using it and the content _should_ be the same as what's in text.md within polaris.shopify.com

Comment (yurm04) [reply polaris-react/src/components/Text/README.md]: yeah i think you're good to delete 👍

**Review by yurm04 — COMMENTED** [review]
@kyledurand the status banner is now set in the markdown content body instead of the frontmatter. [You will need to remove the <StatusBanner> component and message](https://github.com/Shopify/polaris/pull/10381/files#diff-a3f5cd38bb0a30c556c37a99323e76d832477832893cd4bf086aa2dbca38ec32L30-L35) in each file. Also good to remove the `status` frontmatter prop since that's not doing anything anymore I don't think

**Review by yurm04 — APPROVED** [review]
🚢

**Pull request merged**